### ISSUE TITLE
failed to load the converted facenet model.

### ISSUE THREAD

**Issue by vivid9003** [MekkaSiekka/Face-Recognition-with-OpenVino-Toolkit#3]
When I implement this line "executable_network = plugin.LoadNetwork(network, {});"  in "FaceRecognitionClass::initialize(std::string modelfile)", the program will crash. Do you encounter this problem? Or did I do something wrong? @MekkaSiekka

**Comment by MekkaSiekka** [comment]
Could you elaborate what kind of error message it is? Also, have you checked the path to the model file? I used absolute file path in my program so you may need to change the path manually.

**Comment by MekkaSiekka** [comment]
> When I implement this line "executable_network = plugin.LoadNetwork(network, {});" in "FaceRecognitionClass::initialize(std::string modelfile)", the program will crash. Do you encounter this problem? Or did I do something wrong? @MekkaSiekka

Could you elaborate what kind of error message it is? Also, have you checked the path to the model file? I used absolute file path in my program so you may need to change the path manually.

**Comment by vivid9003** [comment]
Yes, the path is correct. These days I found something interesting. The attachments are the original .xml and modified file of the model (20180402-114759, just modified the last 3 output's dimensions from 2 dim to 4 dim). I could successfully load the modified model.
[20180402-114759.zip](https://github.com/MekkaSiekka/Face-Recognition-with-OpenVino-Toolkit/files/2611932/20180402-114759.zip)

**Comment by dliang0406** [comment]
hi @vivid9003, I met exactly the same issue with you. I'm just curious how you found out the solution? I believe it is a bug of MO.
